src/components/Editorial/Ctas.tsx

```diff
@@ -2,22 +2,47 @@ import Button, { type ButtonProps } from '@mui/material/Button';
 import Stack from '@mui/material/Stack';
 import { isValidElement } from 'react';
 import { type CommonProps } from 'types/components';
+import appleBadge from '../../assets/images/app-store-badge.png';
+import googleBadge from '../../assets/images/google-play-badge.png';
+import { ThemeProvider, createTheme } from '@mui/material';
 
 interface CtaButtonProps extends Partial<ButtonProps> {
   text: string;
 }
 
 type CtaButton = CtaButtonProps | JSX.Element;
 
+const customTheme = createTheme({
+  breakpoints: {
+    values: {
+      xs: 0,
+      sm: 420,
+      md: 900,
+      lg: 1200,
+      xl: 1536,
+    },
+  },
+});
```

Comment:
What problem does this custom theme solve? Can the caller change the theme even though it is defined this way here?

Reply:
The issue about this point is for the "sm" breakdown: to adapt the badges, Salvatore needed to change that break to 420 so he did this part as you see. The other breakdowns are not changed from the regular expression but he founds errors if he didn't put them all inside the values.

Reply:
Can the caller change the theme even though it is defined this way here?